pulp_smash/constants.py

```diff
@@ -248,6 +248,11 @@
 )
 """The absolute path to :data:`pulp_smash.constants.RPM` in the filesystem."""
 
+RPM_ENV_DESC = 'A package environment of Pulp tests.'
+
+RPM_ENV_NAME = 'Pulp Test Packages'
+"""The name and description of Package Environments."""
```

Comment:
By the way there is no need to create a constant for the name and description they can go inside the test itself.